misc/export_mappings.py

```diff
@@ -0,0 +1,184 @@
+import argparse
+import logging
+import re
+from bioutils.assemblies import make_name_ac_map
+from contextlib import ExitStack
+from dataclasses import dataclass, field
+
+import psycopg2
+import six
+
+import uta
+
+logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+logger = logging.getLogger("export_mappings")
+
+
+UTA_SCHEMA_VERSION_SQL = """
+select value as schema_version
+from meta
+where key='schema_version';
+"""
+
+ASSOCIATED_ACCESSIONS_SQL = """
+select aa.tx_ac, aa.pro_ac
+from associated_accessions as aa
+where aa.tx_ac='{}';
+"""
+
+# get_tx_mapping_options
+TX_MAPPING_OPTIONS_SQL = """
+select distinct tx_ac,alt_ac,alt_aln_method
+from tx_exon_aln_v where tx_ac='{}' and exon_aln_id is not NULL
+order by alt_ac,alt_aln_method;
+"""
+
+# get_tx_info
+TX_V1_INFO_SQL = """
+select hgnc, cds_start_i, cds_end_i, tx_ac, alt_ac, alt_aln_method
+from transcript T
+join exon_set ES on T.ac=ES.tx_ac
+where tx_ac='{}' and alt_ac='{}' and alt_aln_method='{}';
+"""
+
+TX_V2_INFO_SQL = """
+select G.hgnc, T.cds_start_i, T.cds_end_i, ES.tx_ac, ES.alt_ac, ES.alt_aln_method
+from gene G
+join transcript T on G.gene_id=T.gene_id
+join exon_set ES on T.ac=ES.tx_ac
+where tx_ac='{}' and alt_ac='{}' and alt_aln_method='{}';
+"""
+
+EXON_SET_SQL = """
+select *
+from tx_exon_aln_v
+where tx_ac='{}' and alt_ac='{}' and alt_aln_method='{}'
+order by alt_start_i;
+"""
+
+TX_INDENTITY_SQL = """
+select distinct(tx_ac), alt_ac, alt_aln_method, cds_start_i, cds_end_i, lengths, hgnc
+from tx_def_summary_v
+where tx_ac='{}';
+"""
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Generate alignment metrics for transcript alignments"
```

Comment:
Should the description be updated for this script?

Reply:
Yes, they should!